css-nav-1/Overview.bs

```diff
@@ -1236,37 +1236,69 @@ follow the following steps:
         1. Set <var>candidates</var> be the subset of its items
             whose <a>boundary box</a>'s geometric center is within the closed half plane
             whose boundary goes through the geometric center of the <var>searchOrigin</var>
-            and is perpendicular to <var>D</var>.
+            and is perpendicular to <var>dir</var>.
         2. For each <var>candidate</var> in <var>candidates</var>,
-            find the points <var>P1</var> inside the <a>boundary box</a> of <var>searchOrigin</var>
-            and <var>P2</var> inside the <a>boundary box</a> of <var>candidate</var>
-            that minimize the <var>distance</var> between these two points,
-            when <var>distance</var> is defined as follows:
-
-            <dl>
-              <dt><var>distance</var>:
-                <dd><var>A</var> + <var>B</var> + <var>C</var> - <var>D</var>
-
-              <dt><var>A</var>:
-                <dd>The euclidian distance between <var>P1</var> and <var>P2</var>
-
-              <dt><var>B</var>:
-                <dd>The absolute distance in the <var>dir</var> direction between <var>P1</var> and <var>P2</var>
-
-              <dt><var>C</var>:
-                <dd>The absolute distance in the direction which is orthogonal to <var>dir</var> between <var>P1</var> and <var>P2</var>
-
-              <dt><var>D</var>:
-                <dd>The square root of the area of intersection between the <a>boundary boxes</a> of <var>candidate</var> and <var>searchOrigin</var>
-            </dl>
-        3. Return the item of the <var>candidates</var> set that has the smallest <var>distance</var>.
+            1. If the <var>searchOrigin</var> is an element, then find the points <var>P1</var> inside the <a>boundary box</a> 
```

Comment:
find the points P1 -> find the point P1

Reply:
The word 'points' represents both P1 and P2 (in the next line). Therefore I think the current expression is correct.

Reply:
Agree on it.